crates/voicevox_core/src/infer/runtimes/onnxruntime.rs

```diff
@@ -22,6 +25,16 @@ impl InferenceRuntime for self::blocking::Onnxruntime {
     type Session = ort::Session;
     type RunContext<'a> = OnnxruntimeRunContext<'a>;
 
+    const DISPLAY_NAME: &'static str = {
+        #[cfg(feature = "load-onnxruntime")]
+        {
+            "現在ロードされているONNX Runtime"
+        }
+
+        #[cfg(feature = "link-onnxruntime")]
+        "現在リンクされているONNX Runtime"
```

Comment:
（全然問題ないけど）
書式が違いそう！
```suggestion
        #[cfg(feature = "load-onnxruntime")]
        "現在ロードされているONNX Runtime"

        #[cfg(feature = "link-onnxruntime")]
        "現在リンクされているONNX Runtime"
```

Reply:
それだと文法的に通らないので、書式を合わせるならこう?

```diff
         #[cfg(feature = "load-onnxruntime")]
         {
             "現在ロードされているONNX Runtime"
         }

         #[cfg(feature = "link-onnxruntime")]
-        "現在リンクされているONNX Runtime"
+        {
+            "現在リンクされているONNX Runtime"
+        }
```

ただよく考えたら別のよい書き方に気付いたのでそっちにしました。
[`96ad3cb` (#810)](https://github.com/VOICEVOX/voicevox_core/pull/810/commits/96ad3cbfee707000a64d27adb2bfdfddab9bd3ad)

これができることを失念していた (もう3年前…)
[`panic!` in const contexts - Announcing Rust 1.57.0](https://blog.rust-lang.org/2021/12/02/Rust-1.57.0.html#panic-in-const-contexts)